api/src/pcapi/scripts/move_offer/move_batch_offer.py

```diff
@@ -99,19 +98,33 @@ def _move_individual_offers(origin_venue: offerers_models.Venue, destination_ven
     for offer in origin_venue.offers:
         offer_api.move_offer(offer, destination_venue)
         offer_ids.append(offer.id)
-        offerers_api.create_action_history_when_move_offers(
-            origin_venue.id, destination_venue.id, offer_ids, offers_type="individual"
-        )
+    logger.info(
+        "Individual offers' venue has changed",
+        extra={
+            "origine_venue_id": origin_venue.id,
```

Comment:
Suggestion : à moins que la data n'attende explicitement « origine »,
```suggestion
            "origin_venue_id": origin_venue.id,
```
Idem pour le second log en-dessous.